scripts/check-diff.sh

```diff
@@ -308,7 +308,10 @@ function set_environment_variables {
 		done
 
 		# Make sure linter configs get copied linting directory since upsearch is relative.
-		for linter_file in .jshintrc .jshintignore .jscsrc .jscs.json .eslintignore .eslintrc .eslintrc.js .eslintrc.json .phpcs.xml phpcs.xml .phpcs.xml.dist phpcs.xml.dist phpcs.ruleset.xml ruleset.xml; do
+		#
+		# Eslint might depend on Prettier (https://github.com/prettier/eslint-plugin-prettier), so prettier config has to be copied into the linting directory
+		# Both Eslint and Prettier might be configured as keys in package.json, so this file needs to also be imported
+		for linter_file in .jshint* .jscs* .eslint* package.json .prettier* .phpcs* phpcs.xml phpcs.xml.dist phpcs.ruleset.xml ruleset.xml; do
```

Comment:
Looks like `.phpcs*` already captures `phpcs.xml phpcs.xml.dist phpcs.ruleset.xml` -- should we remove those?

Reply:
Good catch! I've added `phpcs*` (no dot in front of filename): https://github.com/xwp/wp-dev-lib/pull/314/commits/cc46d334a0b5a90da1204950b7942b983c32619f
It could be a single `*phpcs*` though but that seems too greedy 🤔WDYT?